system/user/addons/seo_lite/upd.seo_lite.php

```diff
@@ -199,146 +200,298 @@ function install()
-                'width'		=> '100%',
-                ),   
+                'visible'    => 'true',
+                'collapse'    => 'false',
+                'htmlbuttons'    => 'false',
+                'width'        => '100%',
+            ),
             'seo_lite_twitter_image' => array(
-                'visible'	=> 'true',
-                'collapse'	=> 'false',
-                'htmlbuttons'	=> 'false',
-                'width'		=> '100%',
-                ),   
-            );
+                'visible'    => 'true',
+                'collapse'    => 'false',
+                'htmlbuttons'    => 'false',
+                'width'        => '100%',
+            ),
+        );
 
         return $tabs;
     }
 
 
-	/**
-	 * Uninstall the Seo_lite module
-	 */
-	function uninstall() 
-	{ 				
+    /**
+     * Uninstall the Seo_lite module
+     */
+    function uninstall()
+    {
         $this->EE->load->dbforge();
-        
-		$this->EE->db->select('module_id');
-		$query = $this->EE->db->get_where('modules', array('module_name' => $this->module_name));
-		
-		$this->EE->db->where('module_id', $query->row('module_id'));
-		$this->EE->db->delete('module_member_roles');
-		
-		$this->EE->db->where('module_name', $this->module_name);
-		$this->EE->db->delete('modules');
-		
-		$this->EE->db->where('class', $this->module_name);
-		$this->EE->db->delete('actions');
-		
-		$this->EE->db->where('class', $this->module_name.'_mcp');
-		$this->EE->db->delete('actions');
+
+        $this->EE->db->select('module_id');
+        $query = $this->EE->db->get_where('modules', array('module_name' => $this->module_name));
+
+        $this->EE->db->where('module_id', $query->row('module_id'));
+        $this->EE->db->delete('module_member_roles');
+
+        $this->EE->db->where('module_name', $this->module_name);
+        $this->EE->db->delete('modules');
+
+        $this->EE->db->where('class', $this->module_name);
+        $this->EE->db->delete('actions');
+
+        $this->EE->db->where('class', $this->module_name . '_mcp');
+        $this->EE->db->delete('actions');
 
         $this->EE->dbforge->drop_table('seolite_content');
         $this->EE->dbforge->drop_table('seolite_config');
 
         $this->EE->load->library('layout');
         $this->EE->layout->delete_layout_tabs($this->tabs(), 'seo_lite');
 
-		return TRUE;
-	}
-	
-	/**
-	 * Update the Seo_lite module
-	 * 
-	 * @param $current current version number
-	 * @return boolean indicating whether or not the module was updated 
-	 */
+        return TRUE;
+    }
+
+    /**
+     * Update the Seo_lite module
+     *
+     * @param $current current version number
+     * @return boolean indicating whether or not the module was updated
+     */
     function update($current = '')
     {
-        // NOTHING TO UPDATE YET
+        if ($current == $this->version) {
+            return FALSE;
+        }
+
+        // Update seolite database fields for version 2.0.0
+        if ($current < '2.0.0') {
```

Comment:
Is this the correct version to check for?

Reply:
This seems correct